src/libs/actions/Report.js

```diff
@@ -616,6 +617,29 @@ function createOptimisticChatReport(participantList) {
     };
 }
 
+/*
+ * Creates an optimistic IOU report with a randomly generated reportID
+ */
+function buildOptimisticIOUReport(total, chatReportID, currency, locale) {
```

Comment:
This is not an action per se, so I'd recommend we move it to `src/libs/ReportUtils.js`

Reply:
All the other optimistic creation stuff lives here, though.

Reply:
Yeah, I don't really care either way but seems like keeping it in the Report file for now is fine. Feel free to refactor them all separately though.

Reply:
Resolving, but please reopen if you want me to do this now.